### PR TITLE
Fix staging and prod deploys

### PR DESCRIPTION
**Pull request by mogul**
Since I already ran this in both staging and prod, this should apply without any changes indicated by the plan. The PR just serves to bring the codebase up to date with reality.

### CODE REVIEW

File: application-boundary.tf [hunk 1]
```diff
@@ -63,25 +63,27 @@ resource "cloudfoundry_service_instance" "k8s_cluster" {
...
-#   services      = [cloudfoundry_service_instance.solrcloud_broker_k8s_cluster.id]
-# }
+  name          = "ssb-solrcloud"
+  path          = "./app-solrcloud"
+  broker_space  = var.broker_space
+  client_spaces = var.client_spaces
+  enable_ssh    = var.enable_ssh
+  # services      = [cloudfoundry_service_instance.solrcloud_broker_k8s_cluster.id]
+  services = [data.cloudfoundry_user_provided_service.ssb-solrcloud-k8s.id]
+}
```
Comment (nickumia-reisys) [inline application-boundary.tf:86]: The Solrcloud Broker is available again?

File: application-boundary.tf [hunk 2]
```diff
@@ -63,25 +63,27 @@ resource "cloudfoundry_service_instance" "k8s_cluster" {
 #   ]
 # }
 
-data "cloudfoundry_org" "gsa" {
-  name = var.broker_space.org
+# For now we are using a hand-provisioned user-provided service, not managed by Terraform
+data "cloudfoundry_space" "broker-space" {
+  name     = var.broker_space.space
+  org_name = var.broker_space.org
```
Comment (nickumia-reisys) [inline application-boundary.tf:69]: Is this just to make the ssb more module to be used by others?  Would anyone ever use this repo?  I thought this was specific to us?

**Review by nickumia-reisys — COMMENTED** [review]
I know why this was necessary and I know it takes a ton of effort 🦅 

Just for historical reference, the thing that caused this was the incident on March 4th?

File: application-boundary.tf (hunk 1 above)
Comment (mogul) [reply application-boundary.tf:86]: Yes, using the hand-propped EKS.

**Review by nickumia-reisys — APPROVED** [review]
👓

**Pull request merged**